cmd/peggo/orchestrator.go

```diff
@@ -149,12 +150,12 @@ func getOrchestratorCmd() *cobra.Command {
 				BaseURL: coingeckoAPI,
 			})
 
-			// TODO: figure out where to put this 15% buffer
-			// peggyParams.AverageEthereumBlockTime is in milliseconds. We add a 15% extra as a buffer so txs have time
-			// to get processed.
-			//
-			// Ref: https://github.com/umee-network/peggo/issues/55
-			averageEthBlockTime := time.Duration(peggyParams.AverageEthereumBlockTime/100*115) * time.Millisecond
+			// We multiply the relayer loop multiplier by the ETH block time.
+			// peggyParams.AverageEthereumBlockTime is in milliseconds.
+			relayerLoopDuration := time.Duration(
+				float64(peggyParams.AverageEthereumBlockTime)*
+					konfig.Float64(flagRelayerLoopMultiplier)) *
```

Comment:
1. This is hard to read. I'd opt to put the values into variables and have `relayerLoopDuration` then be a single line.
2. How does this actually work? time.Duration is a int64.

Reply:
1. Ok!
```go
	ethBlockTime := float64(peggyParams.AverageEthereumBlockTime)
	relayerLoopMultiplier := konfig.Float64(flagRelayerLoopMultiplier)
	relayerLoopDuration := time.Duration(ethBlockTime*relayerLoopMultiplier) * time.Millisecond
```

2. So what's happening is that the float gets truncated when casted, so we loose the decimals. But they are milliseconds so we lose as much as 1ms when doing this, so it shouldn't be an issue

Reply:
I see. ACK